rest/route/project_copy.go

```diff
@@ -93,6 +94,10 @@ func (p *projectCopyHandler) Run(ctx context.Context) gimlet.Responder {
 	if err = p.sc.CopyProjectSubscriptions(oldId, projectToCopy.Id); err != nil {
 		return gimlet.MakeJSONErrorResponder(errors.Wrapf(err, "error copying subscriptions from project '%s'", p.oldProject))
 	}
+	//update admins
```

Comment:
Context comments can be really helpful but in this case the comment doesn't say any more than the function name. A more relevant comment could be something like "give new project roles to copied admins."

Also small nit, but: good comment style is to use complete sentences, and to put a space between `//` and the start of the comment.